Looks like dkms 2.0.19 does not pass any environment variables to
the pre-build script and that breaks the build. This patch adapts
the script so that we could set the necessary paths when building
the package.

--- a/pre-build.sh	2017-04-25 08:38:13.000000000 +0300
+++ b/pre-build.sh	2017-05-24 22:45:15.826759306 +0300
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# Workaround for our DKMS:
+SOURCEDIR=@SOURCEDIR@
+BUILDDIR=@BUILDDIR@
+
 KERNELVER=$1
 KERNELVER_BASE=${KERNELVER%%-*}
 
@@ -23,7 +27,9 @@
     [ "$KERNELVER_BASE" = "$oldest" ]
 }
 
-source amd/amdkcl/symbols
+source ${SOURCEDIR}/amd/amdkcl/symbols
+
+cd ${BUILDDIR}
 
 echo '// auto generated by DKMS pre-build.sh' > amd/amdkcl/symbols.c
 for sym in $SYMS; do
@@ -31,7 +37,7 @@
     echo "void *_kcl_$sym = (void *)0x$addr;" >> amd/amdkcl/symbols.c
 done
 
-find ttm -name '*.c' -exec grep EXPORT_SYMBOL {} + \
+find ${SOURCEDIR}/ttm -name '*.c' -exec grep EXPORT_SYMBOL {} + \
     | sort -u \
     | awk -F'[()]' '{print "#define "$2" amd"$2" //"$0}'\
     > include/rename_symbol.h
